### PR TITLE
Reorg

### PR DESCRIPTION
**Pull request by Joeysantoro**
Joey | Tribe (1+1=3) — Today at 6:46 PM
1. rename contract-addresses to protocol-configuration and @addresses to @protocol
Joey | Tribe (1+1=3) — Today at 6:57 PM
2. delete test contract folder (seems unused)
Joey | Tribe (1+1=3) — Today at 7:05 PM
3. delete bondingcurve and related tests
Joey | Tribe (1+1=3) — Today at 7:14 PM
4. move all timelocks from dao and utils into timelocks top level folder
5. rename token dir -> fei 
Joey | Tribe (1+1=3) — Today at 7:26 PM
6. dao folder subfolders for timelock and governor. Move Tribe contracts to tribe folder
7. stabilizer -> peg and delete reserve stabilizers except TribeReserveStabilzer which goes to tribe
8. delete UniswapOracle (unused)
9. CoreRefPausableLib -> libs
10. Delete PassthroughETH (migration utility no longer used)
11. RateLimitedMinter -> fei/minter
klob — Today at 7:52 PM
seems good to me
yay organization
Joey | Tribe (1+1=3) — Today at 7:58 PM
12. Delete TribalChiefSync (v1) (unused)
13. move staking/feirari to fuse/rewards
14. delete RatioPCVController (V1) and StaticPCVDeposit (V1)
almost there 😅

### CODE REVIEW

**Review by ElliotFriedman — DISMISSED** [review]
LGTM!

File: test/integration/tests/psm.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ describe('e2e-peg-stability-module', function () {
     }
   });
 
-  describe('weth-router', async () => {
+  describe.skip('weth-router', async () => {
```
Comment (ElliotFriedman) [inline test/integration/tests/psm.ts:80]: why skip router tests?

Comment (Joeysantoro) [reply test/integration/tests/psm.ts:80]: they were skipped before I think. Can you make a ticket to fix?

Comment (Joeysantoro) [reply test/integration/tests/psm.ts:80]: They were failing here

**Review by ElliotFriedman — APPROVED** [review]

**Pull request merged**